test/torchaudio_unittest/prototype/rnnt_test_impl.py

```diff
@@ -0,0 +1,250 @@
+import torch
+from torchaudio.prototype.models import conformer_rnnt_model
+from torchaudio_unittest.common_utils import TestBaseMixin, torch_script
+
+
+class ConformerRNNTTestImpl(TestBaseMixin):
+    def _get_input_config(self):
```

Comment:
I'd move these helper functions to module-level, so that the the code around the test class is more focused on test logic.